build.gradle

```diff
@@ -123,6 +123,7 @@ dependencies {
 
   compile group: 'io.github.openfeign', name: 'feign-httpclient', version: '9.5.0'
```

Comment:
your story had no information or acceptance criteria on it, so posting it here doesn't give much value, can you add a description and a/c please